templates/express_pg_prisma/controllers/sampleController.ts

```diff
@@ -0,0 +1,41 @@
+import { PrismaClient } from '@prisma/client';
+import { Request, Response } from 'express';
+
+const prisma = new PrismaClient();
+
+export const test = async (req: Request, res: Response) => {
+  res.status(200).json({ message: 'Working' });
+}
+
+export const createUser = async (req: Request, res: Response) => {
+  try {
+    const { name, email } = req.body;
+    const newUser = await prisma.user.create({
+      data: { name, email },
+    });
+    res.status(201).json(newUser);
```

Comment:
Using a `return` will prevent us from this error `Error [ERR_HTTP_HEADERS_SENT]: Cannot set headers after they are sent to the client`.
Another supporting article: https://dev.to/adamkatora/why-you-should-always-use-return-before-ressend-in-express-apis-and-applications-k9k

Here and everywhere below.